include/Primitives/IPrimitive.hpp

```diff
@@ -12,6 +12,7 @@
 #include "Lights/LightsContainer.hpp"
 
 #define IS_HIT(D) ((D >= 0) ? true : false)
+#define IS_INVERSE(X,Y) (((X > 0 && Y < 0) || (X < 0 && Y > 0)) ? true : false)
```

Comment:
Change the name of the Macro to IS_ON_REAR

Reply:
I do not agree with you, we don't say 'rear'

Reply:
In the pr #72, this macro is placed in Vector3D hpp to be used everywhere